packages/slice-machine/server/src/api/pushChanges/checkCustomTypes.ts

```diff
@@ -12,7 +13,10 @@ export function getCustomTypesWithInvalidReferences(
   return localCustomTypes
     .filter((ct) =>
       ct.tabs.some((tab) =>
-        tab.sliceZone?.value.some((z) => !localSliceIds.includes(z.key))
+        tab.sliceZone?.value.some(
+          (z) =>
+            !localSliceIds.includes(z.key) && z.value.type !== SlicesTypes.Slice
```

Comment:
do you think it would be worth making a helper out of it ?
To recognise legacy slices.

Reply:
Fair point, we do use it in a few places.

The helper would just return `slice.type === SlicesTypes.Slice` though

Reply:
I'm out of context but legacy slices are more that just `Slice` just in case. I reviewed a PR on the editor where I talk about that a bit: https://github.com/prismicio/editor/pull/129#discussion_r1106884567